lms/static/scripts/frontend_apps/components/test/SubmitGradeForm-test.js

```diff
@@ -174,31 +167,19 @@ describe('SubmitGradeForm', () => {
       assert.isTrue(
         wrapper
           .find('.SubmitGradeForm__grade-wrapper')
-          .find(FakeSpinner)
-          .props('classNames')
-          .className.includes('is-active')
-      );
-    });
-
-    it('removes the `active` class on the <Spinner> component after the grade has fetched', async () => {
```

Comment:
Did you feel this was redundant?

Reply:
I should have added a note. This wasn't redundant before but it is with the other changes in this PR because the presence of the `is-active` class is used by `isFetchingGrade`.